packages/node-bridge/launcher.js

```diff
@@ -8,10 +8,13 @@ const { Bridge } = require('./bridge.js');
  */
 function makeVercelLauncher(config) {
   const {
+    launcherType = 'Nodejs',
     entrypointPath,
     bridgePath,
     helpersPath,
+    webHandlerPath,
```

Comment:
Since this is a new variable, we can't guarantee it will be defined. Think about legacy Builders and anyone targeting BOA v3.

https://vercel.com/docs/build-output-api/v3#vercel-primitives/serverless-functions/configuration/node-js-config

Reply:
Yes, and it is ok. The launcher code only uses `webHandlerPath` if the incoming launcher type is `EdgeLight`, and having it `undefined` will not create any issue.

The build container is the only one using `makeVercelLauncher`, and will stay consistent: prior versions will not have `webHandlerPath`, and will also forbid `EdgeLight` as launcher type.

Same with BOAv3 compliant apps: since they won't use `EdgeLight`, they won't need `webHandlerPath`.
Let's have a call if I'm missing something.